src/common/SoundSDL.cpp

```diff
@@ -61,10 +61,12 @@ void SoundSDL::read(uint16_t* stream, int length) {
         return;
 
     if (!buffer_size())
+    {
         if (should_wait())
             SDL_SemWait(data_available);
         else
             return;
+    }
```

Comment:
Thanks, somehow I missed this, this should improve stability of the SDL sound driver as well. I rewrote it completely some months ago.

Reply:
This was from here:
```
/home/denisfa/Git/visualboyadvance-m/src/common/SoundSDL.cpp: In member function ‘virtual void SoundSDL::read(uint16_t*, int)’:
/home/denisfa/Git/visualboyadvance-m/src/common/SoundSDL.cpp:63:8: warning: suggest explicit braces to avoid ambiguous ‘else’ [-Wdangling-else]
     if (!buffer_size())
```